manifests/configmap.yaml

```diff
@@ -39,3 +39,4 @@ data:
   pod_terminate_grace_period: 5m
   pdb_name_format: "postgres-{cluster}-pdb"
   eol_node_label: "eol:true"
+  team_api_role_configuration: "log_statement:all"
```

Comment:
Please add  an example of specifying multiple config params. I feel the YAML comment will suffice.

Reply:
I'm reluctant to add anything to this configmap just for the demonstration purposes, as it has been serving a good place to show the operator defaults before and, obviously, I don't want to modify the operator defaults for this particular purpose. Instead, I've added the string with multiple keys to the parameter description in the README.

Reply:
i think that's OK